features/openhab-addons-external/src/main/resources/conf/sonance.cfg

```diff
@@ -0,0 +1,2 @@
+# This sets the refresh interval (in milliseconds) for all instances
+refresh=60000
```

Comment:
please add a `#` in front here as otherwise this value will always be used (and changes done through the Paper UI will be overwritten upon a restart).